designer/server/src/common/helpers/auth/azure-oidc.js

```diff
@@ -138,3 +168,8 @@ export const azureOidcNoop = {
  * @typedef {import('oidc-client-ts').OidcMetadata} OidcMetadata - OpenID Connect (OIDC) metadata
  * @typedef {import('oidc-client-ts').UserProfile} UserProfile - User profile
  */
+
+/**
+ * @typedef {object} idTokenPayload
```

Comment:
I'm torn by this, does the `oidc-client-ts` package have ID token types already?

I'm guessing we've hit custom Microsoft specific territory now 😣

(Does this PR break Keycloak sign in?)

Reply:
Keycloak will probably still work but isn't required anymore, I expect we'll all switch to AAD with a DefraDev account for local development. That way we'll have a more consistent environment between local-dev/dev/test/prod.

I've removed this as it's not required anymore.

Reply:
Ace, thank you

I'd like to test your PR, how do I set up Keycloak to work with the new scopes?

Reply:
I'll give you a ring after standup